### DISCUSSION TITLE
No alerts after updating

### DISCUSSION THREAD

**Discussion by BradMic** [Security-Onion-Solutions/securityonion#13886] [2.4] [answered]
### Version

2.4.110

### Installation Method

Security Onion ISO image

### Description

other (please provide detail below)

### Installation Type

Distributed

### Location

on-prem with Internet access

### Hardware Specs

Meets minimum requirements

### CPU

4

### RAM

16

### Storage for /

130GB

### Storage for /nsm

12TB

### Network Traffic Collection

tap

### Network Traffic Speeds

Less than 1Gbps

### Status

Yes, all services on all nodes are running OK

### Salt Status

No, there are no failures

### Logs

No, there are no additional clues

### Detail

I have not been getting any alerts since updating to v2.4.110.
![image](https://github.com/user-attachments/assets/bf4cf861-75ce-42e9-93c8-e5edf71296a8)

When I go to http://testmynids.org/uid/index.html, no alert is generated.
I do see the traffic with tcpdump on the sensor's monitoring interface.
I can see the pcap files being created on the sensor in /nsm/pcap.
Queries to /detections?q=2100498 shows that the rule is enabled.
Metadata is set to ZEEK.
HOME_NET is set correctly. 
All the nodes in the grid are green.
![image](https://github.com/user-attachments/assets/b31cca19-7573-4294-965b-bdd7418fbd7f)

sudo so-status on run on the sensor show suricate is running.
sudo so-checkin runs on the manager with no failures.
sudo so-elasticsearch-indices-list does NOT show any .ds-logs-suricata-so-* indices.
![image](https://github.com/user-attachments/assets/9147ec0d-3b1a-45ee-abb9-4033dec00344)

I don't know where else to look. Thank you for any pointers.

### Guidelines

- [X] I have read the discussion guidelines at https://github.com/Security-Onion-Solutions/securityonion/discussions/1720 and assert that I have followed the guidelines.

**Comment by rh-ops** [comment]
Try checking your endpoint policies and make sure they aren't applied to the wrong places. If they are correct, then run the following command and provide the output. 

Salt-call pillar.get logstash

**Reply by BradMic** [reply]
I'm not sure what you mean by endpoint policies but there is no output from the command _sudo salt-call pillar.get logstash_ when run on the manager, except for the word local:

The salt minion service is running on all the nodes.

**Comment by BradMic** [comment]
![image](https://github.com/user-attachments/assets/ae125904-b048-48dc-8692-c04fb204a476)

Looks like there was already a highstate running so that's why it showed nothing before. When I ran it again, it seems to have completed successfully.

![image](https://github.com/user-attachments/assets/877c0d4d-0e3d-46bf-9661-dc842d51e39f)

I ran sudo so-test but still no laerts were shown in the SOC Alerts page.

![image](https://github.com/user-attachments/assets/d7db9805-2aca-4a03-a738-751e48267ab9)

Runing the command _sudo docker logs so-filebeat_ on every node returns _Error response from daemon: No such container: so-filebeat_
![image](https://github.com/user-attachments/assets/81ca3f68-bf3d-45c5-893a-03044e43a87e)

Running the command _sudo docker images | grep filebeat_ returns nothing

**Reply by BradMic** [reply]
I just discovered that the elastic agent won't start on the forward node. It's running fine on all the other nodes.
![image](https://github.com/user-attachments/assets/23cf30b9-d675-4811-af75-f8fd29e78395)

**Reply by BradMic** [reply, marked as answer]
So I ended up having to uninstall/reinstall the elastic agent on the sensor, which fixed the issue. The command _/usr/bin/elastic-agent uninstall_ errored out, so I had to manually delete the /opt/Elastic folder and the files /etc/systemd/system/elastic-agent.service, usr/bin/elastic-agent. Then the command _sudo salt-call state.apply elasticfleet.install_agent_grid_ would work to reinstall the elastic agent back on the sensor.

**Comment by rh-ops** [comment]
So now all your issues have been resolved? You're receiving alerts again?

**Reply by BradMic** [reply]
yes

**Reply by NeoLife-iT** [reply]
my new deployment show all node are green but no data received.  Where do you think I should Look?

**Reply by rh-ops** [reply]
> my new deployment show all node are green but no data received. Where do you think I should Look?

Please open a new discussion for this matter.
